int32 location_id
float32 timestamp

---
float32 x
float32 y
float32 z
